packages/taiko-client/bindings/encoding/input.go

```diff
@@ -70,6 +71,89 @@ var (
 			Type: "address",
 		},
 	}
+
+	blockMetadata2Components = []abi.ArgumentMarshaling{
```

Comment:
Let's use blockMetadataV2 to keep the name same with contract

Reply:
Updated